app/models/role.rb

```diff
@@ -134,12 +134,12 @@ def layers
   end
 
   def direct_memberships_dataset(search_options = {})
-    memberships_as_member_dataset.search(search_options)
+    memberships_as_member_dataset.search(**search_options)
       .select(:role_memberships.*)
   end
 
-  def members_dataset(search_options = {})
-    memberships_dataset.search(search_options)
+  def members_dataset(search_options = nil)
```

Comment:
The change does not look compatible
`{}` is an empty hash
`nil` is nil

Reply:
@semyon-estrin ^^

Reply:
Fixed